app/notifications/notifications.component.html

```diff
@@ -2,11 +2,25 @@
   <h1>{{ 'my notifications' | translate }}</h1>
 </md-card>
 <md-card class="default-card">
+  <div class="filter-options" *ngIf="notifications.length != 0">
+    <span>{{ 'FILTER BY' | translate }}</span>
+    <md-select [(ngModel)]="selectedStatus" placeholder="{{ 'Status' | translate }}">
+      <md-option *ngFor="let statusOption of notificationStatusOptions" [value]="statusOption">
+        {{ statusOption | translate }}
+      </md-option>
+    </md-select>
+    <md-select class="select-layout" [(ngModel)]="selectedNotificationType" placeholder="{{ 'Notification types' | translate }}">
```

Comment:
But it's not applied. When I open the notification page, I see no notification, but I should see all notifications.